resume.md

```diff
@@ -1,49 +1,57 @@
-# FIRST LAST NAME
+# Jacqueline Li
+(Canadian citizen with F1 Visa)
 
-(123) 456 - 7890 | email@gmail.com
-
-www.website.com | github.com/myusername | linkedin.com/myusername
+(123) 456 - 7890 | email@gmail.com | linkedin.com/myusername | github.com/myusername
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of California, Berkeley** | BA in Computer Science, Expected Graduation *DEC 2020*
+* **Coursework**:
+Visualization for Design
+The Structure and Interpretation of Computer Programs
+Data Structures
+Information Devices and Systems
+Discrete Mathematics and Probability Theory
+Computer Architecture
+Introduction to Artificial Intelligence
+Computer Security
+Computational Models of Cognition
+Operating Systems and System Programming
+Efficient Algorithms and Intractable Problems
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Languages**: Python ■ Java ■ C ■ SQL ■ LaTeX ■ Go ■ RISC-V ■ Scheme ■ LISP
+* **Version Control**: Git
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Lab Assistant for Introduction to Computer Science**: UC Berkeley | Berkeley, CA *Jan 2018 - May 2018*
+* Guide class of 35 students learning Snap! and Python during 2 hour biweekly labs.
+
+**Robot Master**: Covariant.ai | Berkeley, CA *Mar 2018 - Jun 2019*
+(Covariant.ai is a startup founded by UC Berkeley professor Pieter Abbeel and three UC Berkeley alumni.)
+* Train artificially intelligent robots to grasp, sort, fold, and place different types of objects.
+* Analyze robots’ performance using innovative tools such as home security monitors and spray paint.
+
+**Design Intern**: Method Inc. | New York, NY *Jan 2020*
+* Work in group of two to design ways to present case studies and job requirements.
+* Formulate concepts for illustrations to accompany technology articles.
+
+**Computer Science Tutor** Self | Berkeley, CA  *Nov 2019 - Present*
+* Help students taking computer science courses at UC Berkeley with coding assignments.
```

Comment:
Sounds very similar to your other lab coursework following the theme of tutoring / assisting others.  Can you use this experience to bring up another theme about your career growth?